lib/node_modules/@stdlib/stats/base/dnanvariancetk/examples/index.js

```diff
@@ -18,23 +18,19 @@
 
 'use strict';
 
-var randu = require( '@stdlib/random/base/randu' );
-var round = require( '@stdlib/math/base/special/round' );
-var Float64Array = require( '@stdlib/array/float64' );
+var uniform = require( '@stdlib/random/base/uniform' );
+var filledarrayBy = require( '@stdlib/array/filled-by' );
+var bernoulli = require( '@stdlib/random/base/bernoulli' );
 var dnanvariancetk = require( './../lib' );
 
-var x;
-var i;
-
-x = new Float64Array( 10 );
-for ( i = 0; i < x.length; i++ ) {
-	if ( randu() < 0.2 ) {
-		x[ i ] = NaN;
-	} else {
-		x[ i ] = round( (randu()*100.0) - 50.0 );
+function rand() {
+	if ( bernoulli( 0.8 ) < 1 ) {
+		return NaN;
 	}
+	return uniform( -50.0, 50.0 );
 }
-console.log( x );
+
+var x = filledarrayBy( 10, 'float64', rand );
```

Comment:
```suggestion
var x = filledarrayBy( 10, 'float64', rand );
console.log( x );
```
let's keep the console.log that was originally there